include/knowhere/comp/knowhere_config.h

```diff
@@ -40,6 +40,17 @@ class KnowhereConfig {
     static std::string
     SetSimdType(const SimdType simd_type);
 
+    /**
+     *The purpose of this interface is: part of the sealed indexes default to using bf16 as the base data to achieve
+     *higher capacity; to ensure consistency in computation between growing and sealed, it is necessary to maintain the
+     *same precision in growing calculations as in sealed.
+     */
+    static void
+    EnablePatchForBF16();
```

Comment:
Maybe a better name? Throwing all the context away, solely looking at the name and the comments, I might be confused at what this patch do. `EnableCalculatingFp32AsBf16` or something similar?